lib/io_handler.rb

```diff
@@ -83,12 +71,12 @@ def self.display_draw_message
     print "Game over. Resulted in a draw.\n"
   end
 
-  def self.display_winning_message(player_num)
-    print "Game over. Player #{player_num} won!\n"
+  def self.display_winning_message(player_token)
+    print "Game over. Player #{player_token} won!\n"
   end
 
-  def self.display_message_vs_computer(player_num)
-    if player_num == 1
+  def self.display_message_vs_computer(player_token)
+    if player_token == "X"
```

Comment:
In ruby strings can be mutable.  When you do a comparison like this, it's possible that the two strings contain the the same character, but are not equal.  You are better off using a symbol (`:X` and `:O`) instead of a string.

Reply:
I agree. I will work on using symbols to compare instead of strings (*Note: I did try playing devil's advocate here rather than just accepting your suggestions, but I do remember reading in Well Grounded Rubyist about each string being an object (which means comparing if 2 objects are the same might return false), whereas a symbol will always be the same symbol).